src/Apps/Collect/CollectionAppFixture.ts

```diff
@@ -0,0 +1,40 @@
+import { ArtworksGridEdges } from "Components/ArtworkGrid/__stories__/ArtworkGridFixture"
```

Comment:
We've been putting app-level fixture data in [here](https://github.com/artsy/reaction/tree/master/src/Apps/__test__/Fixtures), but looking at how we currently do it I think co-locating it as you have makes sense. 

@oxaudo brought something up similar today -- thoughts?

Reply:
I like the idea of moving this to a `/Fixtures` directory to centralize those to be reused.

Reply:
Oh yeah noticed this in the apps folder -- it doesn't seem the right place for this one, since this is a component and not an app. We have a similar fixtures folder in the Publishing directory.

Reply:
👍